backend/go-app/main.go

```diff
@@ -1131,9 +1131,9 @@ func checkAdminLogin(resp http.ResponseWriter, request *http.Request) {
 		}
 
 		// No childorg setup, only parent org
-		if len(org.ManagerOrgs) > 0 || len(org.CreatorOrg) > 0 {
```

Comment:
I see that this code skips the user's direct entry into the sub-org through SSO. To be honest, I am not sure if we want the user to log in directly to the sub-org or not. If we do, then commenting out this code won't allow the sub-org to be skipped while checking all orgs. Because of this uncertanity, I comment instead of removing it completely.